dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java

```diff
@@ -42,10 +42,14 @@ public static void main(String[] args) {
 
         @Override
         public void run(String... args) throws Exception {
+            if (dolphinSchedulerManager.schemaIsInitialized()) {
+                dolphinSchedulerManager.upgradeDolphinScheduler();
+                logger.info("upgrade DolphinScheduler finished");
+            } else {
+                dolphinSchedulerManager.initDolphinScheduler();
+            }
             dolphinSchedulerManager.initDolphinScheduler();
             logger.info("init DolphinScheduler finished");
```

Comment:
What about also moving this into the `else` block?

Reply:
Great,I refetched the dev branch,DolphinSchedulerManager class can't import the UpgradeDao.
![image](https://user-images.githubusercontent.com/15651066/147115816-1b8e47db-deaa-4083-81cd-da07b10ae71f.png)

Reply:
Great.Done.